index.js

```diff
@@ -304,10 +304,12 @@ async function fastifyUnderPressure (fastify, opts) {
   }
 }
 
+/* c8 ignore start */
```

Comment:
So many c8 ignore. Why not simply didable coverage all together?

Reply:
How would you do it?
If I understand correctly, it's related to conditions with `monitorEventLoopDelay`.

Reply:
He is saying just leave disable-coverage: true if there are too many lines uncovered

Reply:
Ho, I misunderstood "altogether".

I think ignoring lines that can't be evaluated by CI for well documented reasons is better from completely disabling the coverage. In the first case, you're still forcing developers to think about why some lines aren't covered. 

Is the increase in the number of lines of code the problem?